ui.frontend/src/utils.js

```diff
@@ -296,6 +296,7 @@ class Utils {
                 const _formJson = await HTTPAPILayer.getFormDefinition(_path, _pageLang);
                 console.debug("fetched model json", _formJson);
                 await this.registerCustomFunctions(_formJson.id);
```

Comment:
Are we supporting both clientlib and custom function ? I thought it was either of these two.